docs/organizations/roles-and-permissions-for-synced-organizations.md

```diff
@@ -16,7 +16,7 @@ Depending on your role on the Git provider you will have different permissions o
       <th>Ignore issues and files,<br/>configure code patterns and file extensions,<br/>manage branches</th>
       <th>Configure repository</th>
       <th>Add repository</th>
-      <th>Bulk copy patterns</th>
+      <th>Create and apply coding standards,<br/>Bulk copy patterns</th>
```

Comment:
Check that that the **Coding standards** feature is only available for organization owners/administrators:

https://deploy-preview-823--docs-codacy.netlify.app/organizations/roles-and-permissions-for-synced-organizations/

Reply:
The plan is to be the same as Bulk copy so yes